database/src/models/Restaurant.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
```

Comment:
Is importing mongoose here twice redundant if you can use mongoose.Schema?  It does look cleaner while coding though.

Reply:
Good question - this doesn't technically import mongoose twice, it just pulls the Schema variable off the mongoose object imported on line 1. 

The Mongoose docs always define a Schema variable by pulling it from mongoose.Schema, but the AirBnB styleguide through ESLint complains about that- it seems to always prefer object destructuring. Not sure why they always define Schema as a separate variable - perhaps it's a better practice when using the `new` keyword to always have it on a direct variable, rather than a method. E.g. ```new Schema() vs. new mongoose.Schema()```

 I thought it would be nice to destructure both Schema and model, but model isn't a constructor function and therefore requires a `this` binding, which is why that one is called with mongoose.model.